### ISSUE TITLE
Members section

### ISSUE THREAD

**Issue by gregorywong** [FreeCodeCampHongKong/FreeCodeCampHongKong.github.io#8]
We need a member section which shows the following:
- photo or image
- name
- short description

At a later stage, we should also create a guide to help everyone add themselves to the member section by creating a pull request.

**Comment by anglyeds** [comment]
I am thinking of using Javascript to add user dynamically . 

1) create a function for user profile
2) store the user profile locally(text file, images) at GitHub
3) loop user files details and the images to display every user profile

Question is should be also add roles?? 
3)

**Comment by anglyeds** [comment]
Or we can store it in Json format.

**Comment by gregorywong** [comment]
@anglyeds I was actually thinking of just adding it directly to the page. Of course, the tradeoff would be that the code in the page file becomes much longer, but it might also be
1) faster to do and
2) easier to understand (without needing to understand the extra JavaScript as well).

As for the roles, do you have any ideas about what kinds of roles we should assign? I saw one example that splits people into these three roles:
1. Maintainer
2. Contributor
3. Committer

Reference with more descriptions about each role: https://opensource.guide/leadership-and-governance/

**Comment by anglyeds** [comment]
@gregorywong  yes, now we have ideas, perhaps we bring this to discussion. whoever think that role is suitable for them, just ask them to take it ??

**Comment by gregorywong** [comment]
@anglyeds Yes, you're right. I think it may be better for people to self-assign, as ultimately they know what they're the most comfortable with. Of course, people should be encouraged to leave their comfort zones once in a while ;)

But it seems like this issue actually needs some thought and discussion. Will definitely revisit this issue later!
